src/main/java/com/monumental/controllers/SearchController.java

```diff
@@ -20,11 +20,22 @@
      * This function lets you search monuments using the q query param
      * Ex: GET http://localhost:8080/api/search?q=Memorial
      * TODO: Possibly search related tables such as Tags
-     * @param q The search query string
-     * @return  Matching Monuments based on their title
+     * @param searchQuery The search query string
+     * @return            Matching Monuments based on their title
      */
     @GetMapping("/api/search")
-    public List<Monument> searchMonuments(@RequestParam String q) {
-        return this.monumentService.search(q);
+    @Transactional
```

Comment:
Can you explain what the `@Transactional` annotation does?

Reply:
I think it makes the session stay open for the duration of the function, which I thought I needed to get the tags but because I'm using `.fetch` to get the monuments and their tags in one query it's not necessary so I removed it